scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -73,9 +89,15 @@ def gather_logs(command):
                         required=True, type=valid_krbid)
     parser.add_argument('-c', dest='cluster', help="Cluster name",
                         required=True, choices=VALID_CLUSTER_NAMES)
+    parser.add_argument('-n', dest='nodes', help="Nodes to gather logs from",
```

Comment:
Is it more normal/expected to use: -n node1 node2 node3
Or would the "-n" be repeated:  -n node1 -n node2 -n node3